apm-sniffer/config/agent.config

```diff
@@ -78,3 +78,6 @@ logging.level=${SW_LOGGING_LEVEL:INFO}
 
 # mysql plugin configuration
 # plugin.mysql.trace_sql_parameters=${SW_MYSQL_TRACE_SQL_PARAMETERS:false}
+
+# Kafka producer configuration
+# collector.kafka.bootstrap_servers=${SW_KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
```

Comment:
I think we should share the backend service, and the kafka sender could be activated automatically through the same agent plugin activation way(copy).

Reply:
From the Config, you have implemented the JVM, trace and profile, so, I think when this plugin activated, there is no need to have grpc endpoint, right?